src/preact/context.js

```diff
@@ -59,3 +59,11 @@ export function WithAmpContext(props) {
 
   return <AmpContext.Provider children={props['children']} value={current} />;
 }
+
+/**
+ *
```

Comment:
Might need some help here with typing.

Reply:
This is for now a `JsonObject`. Not for much longer I think, but still the case. I'll ping you when this changes if you haven't yet merged this pull request.

Reply:
Here it is: https://github.com/ampproject/amphtml/pull/30102

Reply:
Thanks Dima sounds good!